build_test.go

```diff
@@ -23,30 +23,16 @@ func TestCommand(t *testing.T) {
 	}
 }
 
-func TestYumUpdate(t *testing.T) {
+func TestInstallPackage(t *testing.T) {
 	t.Parallel()
-	r := thing.NewTestRunner()
+	r := thing.TestNewRunner()
 
-	err := r.UpdateYum()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	wantHistory := []string{"yum update -y"}
-	if !cmp.Equal(wantHistory, r.History) {
-		t.Fatal(cmp.Diff(wantHistory, r.History))
-	}
-}
-
-func TestInstallPackages(t *testing.T) {
-	t.Parallel()
-	r := thing.NewTestRunner()
-
-	err := r.InstallPackages([]string{"epel-release", "ruby"})
+	err := r.InstallPackage([]string{"update", "epel-release", "ruby"})
```

Comment:
So we can take `update` out of this slice now, and we should still see `yum update -y` in the runner's history, shouldn't we, if everything is working.

Reply:
yup, it magically disappeared 👻  😄